protostar/profiler/contract_profiler.py

```diff
@@ -55,11 +50,11 @@ class Instruction:
 class Sample:
     """
     Sample of a resource
-
-    value -- value of a sample (for example 1 memory hole)
-    callstack -- instruction callstack for the reported sample
-    Each instruction has an function assigned so it is easy to deduce the
-    calltree of an functions (pprof format requires samples to be assigned to instruction not functions)
+    Attributes:
```

Comment:
FYI in next PR, you can put docstring for each field after the field. This should be picked by IDEs for example.

```python
class Foo:
    """Class docs"""

    field: int
    """Field docs"""

    moo: bool
    """moo docs"""